usecase/update_test.go

```diff
@@ -0,0 +1,94 @@
+package usecase
```

Comment:
@iasarchi Parabéns pelos testes!
Eles estão limpos, claros e seguindo uma boa estrutura!

Nos próximos testes eu gostaria que você viesse com uma margem para melhoria:

- testes de mesa: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

- padrão AAA: https://brain.d.foundation/Engineering/Backend/Testing+Made+Simple+Best+Practices+for+Golang+Test#:~:text=The%20AAA%20pattern%20provides%20a,variables%20required%20for%20the%20test.

Será que vem uma próxima talk sua @iasarchi ? 👀